src/modelmanager.cpp

```diff
@@ -135,6 +135,17 @@ void ModelManager::logPluginConfiguration() {
             }
             SPDLOG_LOGGER_DEBUG(modelmanager_logger, "Global plugin: {}; key: {}; value: {}", plugin, key, value);
         }
+        allSupportedConfigKeys.insert(
+            allSupportedConfigKeys.end(),
+            std::make_move_iterator(pluginSupportedConfigKeys.begin()),
+            std::make_move_iterator(pluginSupportedConfigKeys.end())
+        );
+    }
+    for (auto& config : pluginConfig) {
+        if (std::find(allSupportedConfigKeys.begin(), allSupportedConfigKeys.end(), config.first) == allSupportedConfigKeys.end())
+        {
+            SPDLOG_LOGGER_WARN(modelmanager_logger, "Plugin config key: {} not found in supported config keys for available devices", config.first);
```

Comment:
Should we just stop at logging warning?
User specified that model should load with specified plugin configuration. Model cannot load with that configuration so we skip it and just print warning?

Either way I am not sure if we shouldn't put similar logic inside model Instance?